data/json/vehicleparts/roof.json

```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "id": "roof_wood",
+    "type": "vehicle_part",
+    "name": "wooden roof",
+    "item": "frame_wood",
+    "color": "brown",
+    "durability": 130,
+    "breaks_into": [ { "item": "splinter", "count": [ 7, 9 ] } ],
+    "mounts": [ { "name": "on_roof", "volume": 20, "weight": 1000 } ],
```

Comment:
A roof is only a flat sheet of mater that keeps the rain out. Installing it only requires such a sheet, so it must be hold in position by the underlying frame.

The frame in turn should carry everything that is mounted "on the roof". The current system allows solar panels, turrets and the like to be installed without a roof. They must therefor be installed on the frame itself, which makes sense (see cloth roof).

A mount point on the roof part itself is probably not needed at all.

> I assume it's a mount point restriction for things that have to stay on when going fast and won't fall off it unless the roof is wrecked.

But that does very much depend on how I connect it. One can weld huge things one a metal roof. Or tie a huge bag onto it.